### ISSUE TITLE
Add new trash types

### ISSUE THREAD

**Issue by krishaamer** [letsdoitworld/World-Cleanup-Day#70]
(building materials, textile, liquid, mixed waste instead of domestic waste, paper and wood separately) - to be discussed through with Knowledge team

**Comment by KadriKalle** [comment]
Hi, just asking for clarification: is the list in the brackets your proposal what to add? what are the current types in the app?

**Comment by KristiinaKerge** [comment]
@KadriKalle yes the list is proposal, collected from pilot countries and from other apps. 
Download the app :) https://play.google.com/store/apps/details?id=com.teeme.ldi&hl=en 
https://itunes.apple.com/us/app/world-cleanup/id1237553057?mt=8

**Comment by KadriKalle** [comment]
my phone still says that it's not compatible with this version (of app), so i cannot download it. i have Samsung Galaxy Nexus 4.3.
We are discussing the waste types with the team.

**Comment by KristiinaKerge** [comment]
plastic, metal, glass, electronics, paper/wood, tyres, domestic waste, furniture, organic waste - in the current list.
https://drive.google.com/open?id=12As8Dn3sChcGU2p2yJcZ6ICrRHo0HghH

**Comment by KadriKalle** [comment]
So, now we have a small problem. Win has suggested to use the Ocean Conservancy's categories, which are quite extensive https://drive.google.com/file/d/1L2oJ2pMt2VcBR_gWeS2JCf0TcbjS3YAY/view?usp=sharing
Point being that Ocean Conservancy is the one who collect most data and we should be able to merge them, so it was suggested to have their categories as standard. From app using perspective, it's clearly a problem, as we cannot have so many categories and it would be uncomfortable to use. So my team's question back to you is: what is the maximum amount of categories we could create?
Enzo also had an idea to maybe create 2 levels of reporting, so that if needed people can have it more in detail. Would that be something doable?

**Comment by KristiinaKerge** [comment]
12 would be doable. Categories that will consist items from OC list, more generic
We can cooperate with Marine Debris Tracker to have detailed list of trash items found during cleanup. It is separate mapping tool for more dedicated waste explorers

**Comment by KadriKalle** [comment]
the conversation about the types is more difficult than you can imagine, as it also ties to what we can do with the data afterwords. so one technical question: can the waste type tabs ("buttons") on the screen be made in different colours? that could help a bit to nudge the way people will click on them.

**Comment by KristiinaKerge** [comment]
@KadriKalle yes the trash type buttons can be in different colours. What is the desired action you want the user to do?

**Comment by KadriKalle** [comment]
the idea we had is maybe we could still make the user silently sort of choose twice, just an example:
household, industrial, construction, commercial (these in one color)
packaging, electronics, biodegradable, textiles, tires, hygiene products, other (these in second color)
We're just trying to find the trade-off point from what can be user friendly and what we need from the data. so we would know little bit about the items, but also where they might have come from.

**Comment by lynxlynxlynx** [comment]
It could be a two step choice, eg. I don't expect anyone to find construction hygiene products, so the option combination ideally wouldn't be there. :stuck_out_tongue:

**Comment by KadriKalle** [comment]
Hi again, our team is still struggling to get to short list of types, but we are almost there. I think it would help us, if you give us a maximum number of characters, so we know what would fit on a screen and what not. I don't know how easy it is for you to estimate that, but I think currently our list is still too long (in terms of long words :P), so that kind of estimate would help us stay in the boundaries. The sooner you can give us, the better :)

**Comment by KadriKalle** [comment]
Just to check one more thing, as it was asked. All the types have to be visible at once on the screen? there is no possibility for drop-down list or anything similar?
The current proposal is still this (having two types of selection, one about the origin and the other about material):

(1) Household, industrial, construction&demolition, cars&vehicles, dangerous

(2) packaging, bottles, other plastics, metals, electronics, biodegradable, clothes/shoes/textiles, tires, hygiene products, other

Is there any possibility to make it all fit somehow?

**Comment by lynxlynxlynx** [comment]
This is how it looks now:
https://lh3.googleusercontent.com/7XLiaSqE3CzoPbQyrJn-nn4zaxNG4LbJRODvjP2eN3D9JB_Z5HPrmEnF6qXmV53dHeI=h900

Of course it's possible to make it longer or two step. Drop downs have poor usability on mobile though, especially if you have to scroll them. Easier to just add another section. 0,02€

**Comment by KristiinaKerge** [comment]
DUX is making designs with two categories (origin and type)

**Comment by kevincrepin** [comment]
Added suggestion for this on Invision.

When creating a trashpoint: https://projects.invisionapp.com/d/main#/console/10062817/275212854/preview

When viewing a trashpoint: https://projects.invisionapp.com/d/main#/console/10062817/275212855/preview#project_console

**Comment by KristiinaKerge** [comment]
New list of types and recommendation to rename Regular and Threat.
From Knowledge team:
In the first stage, where the app asks if the spot is threat or regular, we recommend this wording:
dangerous (toxic, sharp items, etc - requires skilled people)
non-dangerous

Then we come to the types:

(optional to click)
Waste origin:
household
non-household (if we need to give explanations, then this could mean: industries, agriculture, retailers, supermarkets)
construction and demolition
 
 
(mandatory to click)
Waste type:
Glass
Ceramic/bricks
Plastic
Metal
Biological/paper/wood
Textiles/shoes/carpets
Rubber/tires
Electronics
Toxic/chemicals
Car parts/vehicles
Other

**Comment by KristiinaKerge** [comment]
@KadriKalle is it ok if we keep Regular and Threat and add Waste origin and Waste type? the problem is with word "non-dangerous" it is too long

**Comment by KadriKalle** [comment]
can you still keep "dangerous" (toxic, sharp items, etc - requires skilled people)? i think there is also explanation in the current version (can't check, because the app is not working again).
for "non-dangerous" maybe you could use "safe"? or keep "regular" and "dangerous"?

**Comment by lynxlynxlynx** [comment]
Regular and dangerous should be fine.

**Comment by KristiinaKerge** [comment]
@KadriKalle  when user chooses trash origin, it is possible to choose only one option? I mean when user marks household trash origin, he will not be able to mark also a non-household? Is it correct?

**Comment by KristiinaKerge** [comment]
hints what does the origin means:
household - waste from homes, such as food scraps, packaging from food and cleaning products, clothes
non-household - waste coming from shops, factories, often big amount of the same thing (a pile of the same shoe types, large amount of bananas)
construction and demolition - items and materials used in building (bricks, windows, cement, rocks)
